apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs

```diff
@@ -1002,22 +1002,38 @@ private static MutateRowsRequest CreateMutateRowRequest(TableName tableName, IEn
                 },
                 callSettings);
 
-        private BigtableServiceApiSettings UnderlyingClientSettings { get; set; }
+        private protected BigtableServiceApiSettings UnderlyingClientSettings { get; private set; }
+    }
 
-        /// <summary>
-        /// Gets an underlying API client that can be used to make requests.
-        /// </summary>
-        protected abstract BigtableServiceApiClient GetUnderlyingClient();
+    public partial class BigtableClientImpl
```

Comment:
Is this part of the change just merging two partials together?

Reply:
No, nothing got merged. It's just the diff algorithm not realizing which things join up together. Also, I moved the ConvertResult methods from BigtableClient to BigtableClientImpl, so that probably threw it off.